cmflib/cmfquery.py

```diff
@@ -996,6 +998,100 @@ def get_all_executions_by_stage(self, stage_id: int, execution_uuid: t.Optional[
             if execution_uuid in exec_uuid_list:
                 executions_with_uuid.append(execution)
         return executions_with_uuid
+    
+    def create_unique_executions(self, req_info: str, cmd: str, exe_uuid: str) -> str:
```

Comment:
This file have all the methods that query the mlmd file . I dont think this functionality belongs here. May be move to some utils file inside the server folder . Also rename the function name, to reflect the functionality